tests/test_formparsers.py

```diff
@@ -209,7 +209,7 @@ def test_multi_items(tmpdir, test_client_factory):
     with open(path1, "rb") as f1, open(path2, "rb") as f2:
         response = client.post(
             "/",
-            data=[("test1", "abc")],
```

Comment:
Doesn't the future release of httpx also support this syntax?

Reply:
atm this doesn't work.